src/index.ts

```diff
@@ -218,5 +219,12 @@ async function activate(
       createNotebookDiff(model, renderMime, toolbar, translator)
   );
 
+  gitExtension.registerDiffProvider(
+    'ImageDiff',
```

Comment:
@fcollonval What should be the name of the image diff that gets passed here? Is this value arbitrary?

Reply:
Yes